tests/generate_source/build.bp

```diff
@@ -218,11 +218,9 @@ bob_generate_source {
     out: ["output.txt"],
     depfile: true,
     tool: "gen_with_dep.py",
-    cmd: "${tool} -o ${out} -d ${depfile} ${in}",
+    cmd: "${tool} -o ${out} -d ${depfile} ${in} && touch ${gen_dir}/out.h",
```

Comment:
We're testing implicit outputs (which by definition shouldn't be named on the command line).

Can we set things up so that ${tool} writes out.h based on an input flag, so that the test looks closer to the real use-case please

Reply:
Changed in gerrit update.